### DISCUSSION TITLE
Wrong YAML Front Matter when convert `.qmd` to `.ipynb`

### DISCUSSION THREAD

**Discussion by rgaiacs** [quarto-dev/quarto-cli#9818] [Q&A] [answered]
https://quarto.org/docs/tools/jupyter-lab.html#yaml-front-matter says

> The first cell of your notebook should be a Raw cell that contains the document title, author, and any other options you need to specify.

Given `mwe.qmd` as

```
---
title: MWE
---

Lorem ipsum.
```

When I convert `mwe.qmd` into `mwe.ipynb` using `quarto convert mwe.qmd`, I expects that `mwe.ipynb`  will be

```
{
  "cells": [
    {
      "cell_type": "raw",
      "metadata": {},
      "source": [
        "---\n",
        "title: MWE\n",
        "---"
      ]
    },
    {
      "attachments": {},
      "cell_type": "markdown",
      "metadata": {},
      "source": [
        "Lorem ipsum."
      ]
    }
  ],
  "metadata": {
    "kernelspec": {
      "display_name": "Python 3 (ipykernel)",
      "language": "python",
      "name": "python3",
      "path": "/home/raniere/.local/share/jupyter/kernels/python3"
    }
  },
  "nbformat": 4,
  "nbformat_minor": 4
}
```

but, instead, `mwe.ipynb` is

```
{
  "cells": [
    {
      "cell_type": "markdown",
      "metadata": {},
      "source": [
        "---\n",
        "title: MWE\n",
        "---\n",
        "\n",
        "Lorem ipsum."
      ]
    }
  ],
  "metadata": {
    "kernelspec": {
      "name": "python3",
      "language": "python",
      "display_name": "Python 3 (ipykernel)",
      "path": "/home/raniere/.local/share/jupyter/kernels/python3"
    }
  },
  "nbformat": 4,
  "nbformat_minor": 4
}
```

Note that the YAML front matter from the `.qmd` file was converted into a markdown cell instead of a raw cell.

Am I misunderstanding the use of `quarto convert`?

# Environment

`quarto check` produces

```
Quarto 1.5.39
[✓] Checking versions of quarto binary dependencies...
      Pandoc version 3.2.0: OK
      Dart Sass version 1.70.0: OK
      Deno version 1.41.0: OK
      Typst version 0.11.0: OK
[✓] Checking versions of quarto dependencies......OK
[✓] Checking Quarto installation......OK
      Version: 1.5.39
      Path: /opt/quarto/bin

[✓] Checking tools....................OK
      TinyTeX: (not installed)
      Chromium: (not installed)

[✓] Checking LaTeX....................OK
      Using: Installation From Path
      Path: /usr/bin
      Version: 2021

[✓] Checking basic markdown render....OK

[✓] Checking Python 3 installation....OK
      Version: 3.10.12
      Path: /usr/bin/python3
      Jupyter: 5.2.0
      Kernels: python3

[✓] Checking Jupyter engine render....OK

[✓] Checking R installation...........OK
      Version: 4.3.3
      Path: /usr/lib/R
      LibPaths:
        - /home/raniere/R/x86_64-pc-linux-gnu-library/4.3
        - /usr/local/lib/R/site-library
        - /usr/lib/R/site-library
        - /usr/lib/R/library
      knitr: 1.45
      rmarkdown: 2.25

[✓] Checking Knitr engine render......OK
```

**Comment by cderv** [comment, marked as answer]
We do indeed document that first cell with YAML should be raw cell : https://quarto.org/docs/tools/jupyter-lab.html#yaml-front-matter

So it seems like we should do better... 

Though the produced .ipynb is valid and working, right ?

**Reply by rgaiacs** [reply]
> Though the produced .ipynb is valid and working, right ?

Yes, the produced `.ipynb` is valid and it is working.

**Reply by cderv** [reply]
@cscheid do you think we should adapt `quarto convert` to adjust to our raw cell recommandation in our doc ?

**Reply by cscheid** [reply]
We could. I believe we ask that the first cell be `raw` so that our jupyterlab extension can render the metadata nicely in the jupyterlab view. Still, the difference shouldn't affect rendering, as can be checked by converting the two different .ipynb variants back to .qmd.

Let's file an enhancement ticket.

**Reply by cderv** [reply]
Tracked in https://github.com/quarto-dev/quarto-cli/issues/9819
